propagation/usecases.md

```diff
@@ -0,0 +1,131 @@
+
+### Examples
+
+* 
+
+### Solution
+The [APEx Toolbox Cloudfication](toolboxcloud.md) service aims to convert features from desktop based software into 
+cloud-based services that adhere to APEx standards, enabling seamless integration with cloud resources and existing 
+platforms.
+
+### Outcome
+Key features of the desktop software become accessible via openEO or as an OGC Application Package, allowing the project 
+to utilize these features as reusable, on-demand services on existing hosting platforms.
+
+---
+
+## Use Case 2: Improving algorithm performance
+
+### Problem
+A project is experiencing performance issues when running its algorithm.
+
+### Examples
+
+* 
+
+### Solution
+The [APEx Algorithm Enhancement](enhancement.md) service provides support for optimizing your algorithm. Through a structured 
+approach, the algorithm is analyzed and improved for better performance.
+
+### Outcome
+The algorithm's performance is significantly improved.
+
+---
+
+## Use Case 3: Transforming an existing openEO processing graph into an APEx-compliant service and onboarding it on the APEx Algorithm Catalogue
+
+### Problem
+A project has developed an openEO processing graph and seeks to:
+* Enable the service for reuse as an on-demand service.
+* Make it accessible through the APEx Algorithm Catalogue.
+
+### Examples
+
+* 
+
+### Solution
+The [APEx Interoperability and Compliance Guidelines](../interoperability/algohosting.md) provide a starting point for 
+algorithm and workflow developers to understand how to prepare their openEO processing graph for integration with the 
+APEx catalogue. After preparing the processing graph, or if further assistance is required, the 
+[APEx Algorithm Hosting](hosting.md) service can offer additional support with both the implementation of the guidelines 
+and the onboarding process into the APEx Algorithm Catalogue.
+
+### Outcome
+The workflow is successfully onboarded into the APEx Algorithm Catalogue, enhancing its visibility and usability within 
+the EO community as an APEx-compliant service.
+
+---
+
+## Use Case 4: Transforming a non-openEO based workflow into an APEx-compliant service and onboarding it on the APEx Algorithm Catalogue
+
+### Problem
+A project has developed a non-openEO based workflow and seeks to:
+* Enable the service for reuse as an on-demand service.
+* Make it accessible through the APEx Algorithm Catalogue.
+
+### Examples
+
+*
+
+### Solution
+This process requires the workflow developer to select one of the following options:
+* If the workflow is not compatible with cloud-based environments and requires a desktop environment for execution, the 
+project may consider using the [APEx Toolbox Cloudification](toolboxcloud.md) service. This will ensure that the workflow
+is transformed into an APEx-compatible service, enabling its onboarding onto the APEx Algorithm Catalogue through the 
+[APEx Algorithm Hosting](hosting.md) service.
```

Comment:
so worst case, the project has to invoke 2 services right?  The positive side to this perhaps being that there's a checkpoint in between?

Reply:
Indeed to be discussed if this would a good way forward. This would at least make a clear separation of the scope for each service.